RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZT_vect.cc

```diff
@@ -1560,7 +1560,7 @@ void DAClusterizerInZT_vect::dump(const double beta, const vertex_t& y, const tr
 
 void DAClusterizerInZT_vect::fillPSetDescription(edm::ParameterSetDescription& desc) {
   DAClusterizerInZ_vect::fillPSetDescription(desc);
-  desc.add<double>("tmerge", 0.1);            // 4D only
+  desc.add<double>("tmerge", 0.01);           // 4D only
```

Comment:
this value is incorrect now, if it is not anymore overwritten by a python. And could explain the observed differences in the PU test. I am going to fix it.